src/Microsoft.TemplateEngine.Cli/Dotnet.cs

```diff
@@ -134,40 +165,14 @@ private void ForwardStreamStdErr(object sender, DataReceivedEventArgs e)
             Console.Error.WriteLine(e.Data);
         }
 
-        public Dotnet CaptureStdOut()
-        {
-            _stdout = new StringBuilder();
-            _outputDataReceived += CaptureStreamStdOut;
-            return this;
-        }
-
         private void CaptureStreamStdOut(object sender, DataReceivedEventArgs e)
         {
-            _stdout.AppendLine(e.Data);
-        }
-
-        public Dotnet CaptureStdErr()
-        {
-            _stderr = new StringBuilder();
-            _errorDataReceived += CaptureStreamStdErr;
-            return this;
+            _stdout?.AppendLine(e.Data);
         }
 
         private void CaptureStreamStdErr(object sender, DataReceivedEventArgs e)
         {
-            _stderr.AppendLine(e.Data);
-        }
-
-        public Result Execute()
-        {
-            Process p = Process.Start(_info);
```

Comment:
Are we removing some functionality or is this just a member order change?

Reply:
That's order change